apiary.apib

```diff
@@ -332,24 +332,32 @@ Get the current authenticated user.
             + last_name: `Smith` (string)
             + email: `john@teamleader.eu` (string)
             + telephones (array[Telephone])
-            + language: `nl-BE` (enum[string])
-                + Members
-                    + da-DK
-                    + de-DE
-                    + en-GB
-                    + es-ES
-                    + fi-FI
-                    + fr-FR
-                    + it-IT
-                    + nb-NO
-                    + nl-BE
-                    + nl-NL
-                    + pl-PL
-                    + pt-PT
-                    + sv-SE
-                    + tr-TR
             + function: `Sales` (string)
-            + time_zone: `Europe/Brussels` (string)
+            + preferences (object)
+                + language: `nl-BE` (enum[string])
+                    + Members
```

Comment:
Maybe not 100% in scope with this PR, but:

I'm not sure if the list is actually correct. Most of the languages are not country specific in Teamleader.
Spanish for example, is just `es` and not `es-ES`.
We actually only have `fr-FR` and `nl-NL` ( where `fr` and `nl` actually resemble `fr-BE` and `nl-BE`).

It's also missing 2 values that are available in the UI (`fr` and `ca`).

Reply:
We chose follow the ISO "locale" standard here. We shouldn't return a mix of countries and locales.

The missing values are out of scope though, those can be addressed in a different PR.